playbooks/common/openshift-cluster/evaluate_groups.yml

```diff
@@ -67,6 +67,15 @@
     when: g_master_hosts|length > 0
     changed_when: no
 
+  - name: Evaluate oo_new_etcd_to_config
+    add_host:
```

Comment:
You could move this task into `playbooks/common/openshift-cluster/evalute_groups.yml` file which calls `add_host` for various groups.